modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/folders/SuggestedFoldersPage.kt

```diff
@@ -0,0 +1,160 @@
+    onCreateCustomFolders: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier.fillMaxSize(),
+    ) {
+        IconButton(
+            onClick = onDismiss,
+        ) {
+            Icon(
+                painter = painterResource(IR.drawable.ic_close),
+                contentDescription = stringResource(LR.string.close),
+                tint = MaterialTheme.theme.colors.primaryInteractive01,
+                modifier = Modifier.padding(16.dp),
+            )
+        }
+
+        Column(
+            modifier = Modifier
+                .padding(horizontal = 16.dp)
+                .padding(bottom = 16.dp),
+        ) {
+            TextH10(
+                text = stringResource(LR.string.suggested_folders),
+                lineHeight = 36.sp,
+                modifier = Modifier.padding(bottom = 8.dp),
+            )
+
+            TextP40(
+                text = stringResource(LR.string.suggested_folders_subtitle),
+                color = MaterialTheme.theme.colors.primaryText02,
+                modifier = Modifier.padding(bottom = 16.dp),
+            )
+
+            Folders(
+                podcastUuids = mockedPodcastsUuids,
+                Modifier
+                    .padding(bottom = 8.dp)
+                    .weight(1f),
+            )
+
+            RowButton(
+                text = stringResource(LR.string.suggested_folders_use_these_folders_button),
+                modifier = Modifier.padding(bottom = 16.dp),
+                textColor = MaterialTheme.theme.colors.primaryInteractive02,
+                fontSize = 18.sp,
+                fontWeight = FontWeight.W600,
+                colors = ButtonDefaults.buttonColors(
+                    backgroundColor = MaterialTheme.theme.colors.primaryInteractive01,
+                ),
+                includePadding = false,
+                onClick = onUseTheseFolders,
+            )
+
+            RowOutlinedButton(
+                text = stringResource(id = LR.string.suggested_folders_use_create_custom_folders_button),
+                onClick = onCreateCustomFolders,
+                includePadding = false,
+                colors = ButtonDefaults.outlinedButtonColors(contentColor = MaterialTheme.theme.colors.primaryIcon01, backgroundColor = Color.Transparent),
+                fontSize = 18.sp,
+                fontWeight = FontWeight.W600,
+            )
+        }
+    }
+}
+
+@Composable
+private fun Folders(podcastUuids: List<String>, modifier: Modifier = Modifier) {
+    val folders = 4
+
+    LazyVerticalGrid(
+        columns = GridCells.Adaptive(110.dp),
+        horizontalArrangement = Arrangement.spacedBy(6.dp),
+        verticalArrangement = Arrangement.spacedBy(6.dp),
+        modifier = modifier,
+    ) {
+        items(
+            count = folders,
+            key = { index -> index },
+        ) { index ->
+            FolderItem("Test", Color.Yellow, podcastUuids)
+        }
+    }
+}
```

Comment:
Maybe we should include the title and description in the scroll view to make it slightly better in landscape. 

![Screenshot_20250206_133922](https://github.com/user-attachments/assets/db2296f1-c261-4ab2-bf03-35eb4b3ae4cc)